codec_test.go

```diff
@@ -49,3 +49,47 @@ func TestCodec(t *testing.T) {
 	require.Equal(t, math.MaxInt32, targetInt, "expected codec to successfully decode int")
 	require.Equal(t, 0, len(buf.Bytes()), "expected buffer to be empty")
 }
+
+func TestMap(t *testing.T) {
+	var buf bytes.Buffer
+	mh := &codec.MsgpackHandle{WriteExt: true}
```

Comment:
is this different from the MsgpackHandles used in other places on purpose? (i.e., not having RawToString set to true)